### ISSUE TITLE
Reconnect to the server as soon as the app is opened (AirMessage Cloud)

### ISSUE THREAD

**Issue by tagavari** [airmessage/airmessage-android#43]
When using AirMessage Cloud, the app will not maintain a connection in the background. This can cause issues when the app is restored to the foreground, where a connection to the server will not be restored without explicit user interaction.

Instead, the app should restore its connection to the server automatically when brought to the foreground.

Reported by Avibee and others on Discord: https://discord.com/channels/992925221256450068/992929582439202908/1004535441212571769

**Comment by aviavi123** [comment]
Yes, this happens to me, as noted in here https://github.com/airmessage/airmessage-server/issues/27
